examples/omni-lock-metamask/package.json

```
@@ -10,7 +10,7 @@
   "author": "",
   "license": "MIT",
   "dependencies": {
-    "@ckb-lumos/lumos": "0.19.0-alpha.3",
+    "@ckb-lumos/lumos": "0.19.0-beta.0",
```

```suggestion
    "@ckb-lumos/lumos": "0.19.0-alpha.3",
```

I found the beta branch is not merged into develop yet, so maybe we don't need to change the dep version now

https://github.com/ckb-js/lumos/blob/next%2Fv0.19.0-beta.0/examples/omni-lock-metamask/package.json

---

But in `0.19.0-alpha3`, the module do not export `omni` module.

https://github.com/ckb-js/lumos/blob/v0.19.0-alpha.3/packages/common-scripts/src/index.ts#L13

---

bumped develop to beta.0, we can rm this line, this line would make yarn install lumos from npm instead of using the workspace's lumos